public/docs/ts/latest/guide/architecture.jade

```diff
@@ -53,14 +53,14 @@ figure
   Angular apps are modular and Angular has its own modularity system called _Angular Modules_ or _NgModules_.
 
   _Angular Modules_ are a big deal. 
-  We can only introduce them here; the [Angular Modules](ngmodule.html) chapter covers modules in depth.
+  This is only an introduction; the [Angular Modules](ngmodule.html) chapter covers modules in depth.
 
 <br clear="all"><br>
 :marked
   Every Angular app has at least one module, the _root module_, conventionally named `AppModule`.
 
-  While the _root_ module may be the only module in a small application, most apps have many more 
-  _feature_ modules, each a cohesive block of code dedicated to an application domain,
+  While the _root module_ may be the only module in a small application, most apps have many more 
+  _feature modules_, each a cohesive block of code dedicated to an application domain,
   a workflow, or a closely-related set of capabilities. 
```

Comment:
closely-related -> closely related
(per http://www.cjr.org/language_corner/sincere-ly_yours.php, if I'm reading it right)